python-checks/src/test/resources/checks/django/djangoModelStringFieldCheck.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+class NullFieldsModel(models.Model):
+    name = models.CharField(max_length=50, null=True) # Noncompliant {{Replace this "null=True" flag with "blank=True".}}
+#                                          ^^^^^^^^^ 0
+    desc = models.TextField(max_length=50, null=True) # Noncompliant
+
+class NullBlankFieldsModel(models.Model):
+    name = models.CharField(max_length=50, null=True, blank=True) # Noncompliant {{Remove this "null=True" flag.}}
+    desc = models.TextField(max_length=50, null=True, blank=True) # Noncompliant
+
+class BlankFieldsModel(models.Model):
+    name = models.CharField(max_length=50, blank=True)
+    desc = models.TextField(max_length=50, blank=True)
+
+
```

Comment:
One case that seems to be missing is when both `unique=True` and `blank=True`, where we should not raise an issue.

Reply:
done